File: app/src/main/java/com/chuongvd/app/signal/data/database/model/Signal.java
package com.chuongvd.app.signal.data.database.model;

public interface Signal {
    int getId();
    String getSymbol();
    long getDateIn();
    String getType();
    float getPriceIn();
    float getTakeProfitPrice();
    float getStopLossPrice();
    String getStatus();
}

File: app/src/main/java/com/chuongvd/app/signal/data/database/model/Signal.kt
package com.chuongvd.app.signal.data.database.model

interface Signal {
  val id: Int
  val symbol: String?
  val dateIn: Long
  val type: String?
  val priceIn: Float
  val takeProfitPrice: Float
  val stopLossPrice: Float
  val status: String?
}
